modules/RTC/RTC.js

```diff
@@ -151,7 +152,12 @@ export default class RTC extends Listenable {
         this._updateAudioOutputForAudioTracks
             = this._updateAudioOutputForAudioTracks.bind(this);
 
-        // The default video type assumed by the bridge.
+        /**
+         * The default video type assumed by the bridge.
+         * @deprecated this will go away with multiple streams support
+         * @type {BridgeVideoType}
+         * @private
+         */
```

Comment:
the default type assumed by the JVB is actually CAMERA, but I'm a bit afraid to touch it. This value belong to the existing flow and is not used with the multi stream support